packages/application-generic/src/usecases/add-job/add-digest-job.usecase.ts

```diff
@@ -32,9 +32,11 @@ type AddDigestJobResult = number | undefined;
 @Injectable()
 export class AddDigestJob {
   constructor(
+    @Inject(forwardRef(() => EventsDistributedLockService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.